windows-containers/windows-monitoring.md

```diff
@@ -0,0 +1,165 @@
+---
+title: OpenShift Windows Worker Node Monitoring
+authors:
+  - "@ravisantoshgudimetla"
+  - "@aravindhp"
+reviewers:
+  - "@crawford"
+  - "@sdodson"
+  - "@brancz"
+  - "@s-urbaniak"
+  - "@lilic"
+approvers:
+  - "@crawford"
+  - "@sdodson"
+  - "@brancz"
+  - "@s-urbaniak"
+  - "@lilic"
+creation-date: 2019-09-17
+last-updated: 2019-09-18
+status: implementable
+---
+
+# monitoring-of-windows-worker-nodes
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+The intent of this enhancement is to enable monitoring of Windows worker nodes
+in OpenShift cluster.
+
+## Motivation
+
+Monitoring is critical to identify issues with nodes, containers running on the
+nodes. The main motivation behind this enhacement is to enable monitoring of
+Windows nodes
+
+### Goals
+
+As part of this enhacement, we plan to do the following:
+* Deploy Windows Management Instrumentation(WMI exporter) onto Windows nodes
+* Configure Prometheus provisioned as part of OpenShift install to 
```

Comment:
With `Configure Prometheus` do you mean the deployment of a service monitor as explained below?

Reply:
Yes, that's what I meant. Please let me know, if this needs clarification.

Reply:
this is fine from my perspective :+1: